src/content/ja/2021/performance.md

```diff
@@ -0,0 +1,509 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: パフォーマンス
+description: 2021年版Web Almanacのパフォーマンス編では、Core Web Vitals（最大コンテンツの描画、累積レイアウトシフト、初回入力遅延）に加え、コンテンツの初回ペイントと最初のバイトまでの時間をカバーしています。
+authors: [siakaramalegos]
+reviewers: [rviscomi, kevinfarrugia, estelle, ziemek-bucko, jzyang, fili, tunetheweb, samarpanda, edmondwwchan]
+analysts: [siakaramalegos, rviscomi, Nithanaroy]
+editors: [jzyang]
+translators: [ksakae1216]
+results: https://docs.google.com/spreadsheets/d/13xhPx6o2Nowz_3b3_5ojiF_mY3Lhs25auBKM6eqGZmo/
+siakaramalegos_bio: Sia Karamalegosはウェブ開発者、国際会議のスピーカー、ライターです。Googleデベロッパー ウェブテクノロジーの専門家、Cloudinaryメディア開発者エキスパート、Stripeコミュニティエキスパートであり、Eleventy Meetupを共同開催しています。<a hreflang="en" href="https://sia.codes/">sia.codes</a> や<a hreflang="en" href="https://twitter.com/thegreengreek">Twitter</a> で、彼女の執筆、講演、ニュースレターをチェックしてください。
+featured_quote: フレームワークレベルでパフォーマンスに関するスマートなデフォルトを設定できれば、より良いウェブを作ることができ、同時に開発者の仕事を楽にすることができます。
+featured_stat_1: 37%
+featured_stat_label_1: 上位1,000サイトのうちCore Web Vitalsを通過したサイトの割合
+featured_stat_2: 79%
+featured_stat_label_2: デスクトップで画像をLCP要素として表示したページ
+featured_stat_3: 67 s
+featured_stat_label_3: 最長の総ブロッキング時間
+---
+
+## 序章
+
+ユーザー体験にとって、パフォーマンスは重要です。読み込みが遅く、反応が遅いWebサイトは、ユーザーをイライラさせ、コンバージョンの低下を招きます。<a hreflang="en" href="https://web.dev/vitals/">Core Web Vitals</a>が<a hreflang="en" href="https://developers.google.com/search/blog/2020/11/timing-for-page-experience">Google検索ランキング</a>に寄与するのは今年がはじめてです。そのため、ウェブサイトのパフォーマンス向上に対する関心が高まっており、ユーザーにとっては嬉しいニュースです。
```

Comment:
web.dev has started to become avaialble in Japanese!

```suggestion
ユーザー体験にとって、パフォーマンスは重要です。読み込みが遅く、反応が遅いWebサイトは、ユーザーをイライラさせ、コンバージョンの低下を招きます。<a hreflang="ja" href="https://web.dev/i18n/ja/vitals/">Core Web Vitals</a>が<a hreflang="en" href="https://developers.google.com/search/blog/2020/11/timing-for-page-experience">Google検索ランキング</a>に寄与するのは今年がはじめてです。そのため、ウェブサイトのパフォーマンス向上に対する関心が高まっており、ユーザーにとっては嬉しいニュースです。
```

Could you check the rest of these web.dev links?

Reply:
I didn't know.
Thanks, I translated.